README.md

```diff
@@ -284,7 +284,7 @@ class Foo
 
   # use changes_with_baz to include baz's changes in this document's
   # history.
-  track_history     :changes_method => :changes_with_baz
+  track_history   :on => fields.keys + [:baz], :changes_method => :changes_with_baz
```

Comment:
You should add a section to the README called tracking custom fields and describe this in more detail, explaining the default, with examples of how to track specific fields and how to add non-fields to be tracked.